pyopenms_viz/_dataframe.py

```
@@ -0,0 +1,141 @@
+        """Get minimum value of a column."""
+        if self._is_polars:
+            return self._data[col].min()
+        return self._data[col].min()
+    
+    def iterrows(self):
+        """Iterate over DataFrame rows."""
+        if self._is_polars:
+            for row in self._data.iter_rows(named=True):
+                yield row
+        else:
+            for idx, row in self._data.iterrows():
```

Ruff (0.8.2)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)